skbio/stats/composition.py

```diff
@@ -150,8 +158,9 @@ def multiplicative_replacement(mat, delta=None):
     delta: float, optional
        a small number to be used to replace zeros
        If delta is not specified, then the default delta is
-       :math:`\delta = \frac{1}{N^2}` where :math:`N`
-       is the number of components
+       :math:`\delta = \frac{1}{N M}` where :math:`N`
+       is the number of components and :math:`M` is the
+       number of individuals found in the largest sample
```

Comment:
Oh ....

Crap.  That was poor oversight.  I remember why I used `delta=N^2` before.  This was appropriate for proportions.  The `delta=1/NM` is appropriate for counts.  I'm going to revert ...